docs/check.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM onosproject/onos-docs-base:latest
```

Comment:
Can't see why it needs another Docker image - couldn't it be executed on the build image, with just a different entrypoint?

Reply:
It should be possible. I was thinking perhaps it would be better to make them separate in case we want to update just the checker with more stuff but you are right. For now it would be better to have one image.